services/src/tests/e2e/tests/__snapshots__/authorization-with-queries.spec.ts.snap

```diff
@@ -10,8 +10,8 @@ Object {
         "exception": Object {
           "stacktrace": Array [
             "Error: Unauthorized by policy alwaysDenied in namespace namespace",
-            "    at Function.validatePolicy (/service/dist/modules/directives/policy/policy-executor.js:27:19)",
-            "    at async field.resolve (/service/dist/modules/directives/policy/policy.js:13:17)",
+            "    at Function.validatePolicy (/service/dist/modules/directives/policy/policy-executor.js:28:19)",
```

Comment:
Not in this PR, but we should do something about the snapshots.
It's unreasonable to have to change this for any internal code change that has no effect on the outcome.

Do the snapshots support partial fuzzy matching for sections of the result?

Reply:
Who cares? They are autogenerated, just run em and they fix themselves
It's not ideal but it's really not worth investing time into IMO

Reply:
How do you generate them? I manually had to copy the changes from the error about snapshot mismatch and replace it in the snapshot. I didn't see anything about auto-generating them

Reply:
https://jestjs.io/docs/en/snapshot-testing#updating-snapshots

Reply:
I'll change the snapshot not to include stack trace in another PR